service/commentutil/commentutil.go

```diff
@@ -39,6 +40,14 @@ func (p PostedComments) AddPostedComment(path string, lineNum int, body string)
 	p[path][lineNum] = append(p[path][lineNum], body)
 }
 
+func (p PostedComments) DebugLog() {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method PostedComments.DebugLog should have comment or be unexported